Formula/gdal.rb

```diff
@@ -1,9 +1,9 @@
 class Gdal < Formula
   desc "Geospatial Data Abstraction Library"
   homepage "https://www.gdal.org/"
-  url "https://download.osgeo.org/gdal/2.4.0/gdal-2.4.0.tar.xz"
-  sha256 "c3791dcc6d37e59f6efa86e2df2a55a4485237b0a48e330ae08949f0cdf00f27"
-  revision 3
+  url "https://download.osgeo.org/gdal/2.4.1/gdal-2.4.1.tar.xz"
+  sha256 "fd51b4900b2fc49b98d8714f55fc8a78ebfd07218357f93fb796791115a5a1ad"
+  revision 1
```

Comment:
You don't need `revision 1` when the `version` increases.

Reply:
Fixed. Thanks!